aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -271,49 +216,18 @@ In the preceding example:
 
 ## Publish to an MSDeploy endpoint from the command line
 
-The following example uses an ASP.NET Core web app created by Visual Studio named *AzureWebApp*. An Azure Apps publish profile is added with Visual Studio. For more information on how to create a profile, see the [Publish profiles](#publish-profiles) section.
-
-To deploy the app using a publish profile, execute the `msbuild` command from a Visual Studio **Developer Command Prompt**. The command prompt is available in the *Visual Studio* folder of the **Start** menu on the Windows taskbar. For easier access, you can add the command prompt to the **Tools** menu in Visual Studio. For more information, see [Developer Command Prompt for Visual Studio](/dotnet/framework/tools/developer-command-prompt-for-vs#run-the-command-prompt-from-inside-visual-studio).
+See [Microsoft.NET.Sdk.Publish](https://github.com/dotnet/sdk/tree/main/src/WebSdk#microsoftnetsdkpublish).
 
-MSBuild uses the following command syntax:
-
-```bash
-msbuild {PATH} 
-    /p:DeployOnBuild=true 
-    /p:PublishProfile={PROFILE} 
-    /p:Username={USERNAME} 
-    /p:Password={PASSWORD}
-```
+<!-- As of Nov 2024 
```

Comment:
Why is this commented out?

Reply:
That's internal for us to test fetching PW. I don't want publish how to get and save the pw, it's not hard to figure out.